contributors/design-proposals/node/troubleshoot-running-pods.md

```diff
@@ -387,14 +407,10 @@ basic functionality:
 *   `kubectl describe pod` will list status of debug containers running in a pod
 
 Functionality will be hidden behind an alpha feature flag and disabled by
-default. The following are explicitly out of scope for the 1.9 alpha release:
+default. The following are explicitly out of scope for the 1.10 alpha release:
 
 *   Exited Debug Containers will be garbage collected as regular containers and
     may disappear from the list of Debug Container Statuses.
```

Comment:
> Exited Debug Containers will be garbage collected as regular containers and may disappear from the list of Debug Container Statuses.

harmonize this with the "The list of `EphemeralContainerStatuses` is never truncated." statement below